.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Tests
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 14, 16, 18 ]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm i
+      - run: npx eslint . --report-unused-disable-directives
```

Comment:
Adding the flag ensures that `eslint-disable-next-line` are reported as false positive failures if the following line does not in fact trigger the rule